README.md

```diff
@@ -113,6 +113,42 @@ ResponseCode response = SlackWebHookService.with(token)
 RoboSlack handles error responses through the ``ResponseCode`` class. Check your ``ResponseCode`` to see why a message
 may have failed to send.
 
+#### Slack Date Formatting
+
+RoboSlack supports Slack's Date formatting in fields that allow Slack Markdown. 
+See [the "Formatting Dates" section in the Slack Docs for more information](https://api.slack.com/docs/message-formatting).
+
+```
+// Simple Date Formatting example
+MessageRequest message = MessageRequest.builder()
+                .username("roboslack")
+                .text(String.format("date_short_pretty: %s",
+                    SlackDateTime.create(ZonedDateTime.now(), DateTimeFormatToken.DATE_SHORT_PRETTY)))
+                .build();
+```
+
+Using the `SlackDateTime.create()` method, input a Java `Temporal` and a `DateTimeFormatToken` that represents how you'd
+like to see your Date rendered in your Slack message.
+
+Make sure that you specify a `Temporal` that contains all of the necessary Time/Date fields for proper rendering 
+according to this table:
+
+
+|Format Name        | Required Temporal Fields| Description                                                                           |
```

Comment:
This is an awesome idea! I love this table!  What if we added a column for what implementors of a `Temporal` contain the correct information for each token?  That way it will serve as a good reference point for these things going forward.

What about also explaining what happens when you provide a `Temporal` type that doesn't contain the correct information?